src/components/navigation/perPage/NavigationPerPageReducers.ts

```diff
@@ -0,0 +1,50 @@
+import { IReduxAction, IReduxActionPayload } from '../../../utils/ReduxUtils';
+import { PerPageActions } from './NavigationPerPageActions';
+import * as _ from 'underscore';
+
+export interface IPerPageState {
+  id: string;
+  perPage: number;
+}
+
+export const perPageInitialState: IPerPageState = {
+  id: undefined,
+  perPage: 10
+};
+
+export const perPageCompositeInitialState: IPerPageState[] = [];
+
+export const perPageReducer = (state: IPerPageState = perPageInitialState, action: IReduxAction<IReduxActionPayload>): IPerPageState => {
```

Comment:
If this interface is necessary for UTs, why not simply put it in UTs where you need it? For me the ReduxUtils.ts file is intended to regroup base interfaces/functions useful to connect components and use them.